modules/social_features/social_like/social_like.module

```diff
@@ -88,5 +88,8 @@ function social_like_preprocess_like_and_dislike_icons(&$variables) {
       }
     }
   }
-  $variables['modal_title'] = t('Members who liked this @content', ['@content' => $bundle]);
+  $variables['modal_title'] = t('Members who like this @content', ['@content' => $bundle]);
+  if ($variables['entity_type'] == 'post') {
```

Comment:
@Macsym2018 please add explanation comment why we adding special case for Posts.
You can take it from the PR Problem and re-phrase a bit.

Reply:
@ribel Done